assets/js/translator.js

```diff
@@ -4,10 +4,39 @@ var curSrcLang, curDstLang;
 var recentSrcLangs = [], recentDstLangs = [];
 var droppedFile;
 var textTranslateRequest;
+var recaptchaScriptSrc = "https://www.google.com/recaptcha/api.js?onload=recaptchaRenderCallback&render=explicit&hl=";
+
+var recaptchaRenderCallback = function () {
```

Comment:
Why not...? It's still in global namespace if placed directly under L15.
Just remove the var or use window.(variable name). In fact I'd recommend
placing it right after you check if suggestions are enabled.

On Aug 25, 2016 7:38 AM, "Kira Droganova" <notifications@github.com> wrote:

> In assets/js/translator.js
> <https://github.com/goavki/apertium-html-tools/pull/68#discussion_r76234239>
> :
>
> > @@ -4,10 +4,39 @@ var curSrcLang, curDstLang;
> >  var recentSrcLangs = [], recentDstLangs = [];
> >  var droppedFile;
> >  var textTranslateRequest;
> > +var recaptchaScriptSrc = "https://www.google.com/recaptcha/api.js?onload=recaptchaRenderCallback&render=explicit&hl=";
> > +
> > +var recaptchaRenderCallback = function () {
>
> It seems we can't.
> It does not want to render if not in global namespace.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/goavki/apertium-html-tools/pull/68/files/f4ebebee15e2453007e977dc5a3753dc0ea65f92#r76234239>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEBEfmWvrCBVsVoDJsXqetB153kAPOdXks5qjYzWgaJpZM4JpRx5>
> .
>